Dashboard does not work

<!-- Love localstack? Please consider supporting our collective:
👉  https://opencollective.com/localstack/donate -->
Run localstack through docker. 
```bash
sudo docker run -it -p 4567-4578:4567-4578 -p 8080:8080 localstack/localstack
```
I have put some objects into s3. 
```bash
$ aws s3 ls foo.bucket
2019-06-22 15:15:02         11 bar.key
```
But the dashboard is still empty.  The dashboard keeps empty even if I add lambda functions and more objects.

Experiencing the same issues. 

The files causing the initial errors are actually on the docker image (tests?) e.g. 
/tmp/cache.a743de848abda9c36cceb4392861a8f6.json

chowning them to localstack:localstack gets rid of the error in the log and the cache gets subsequently cleared when refreshing the UI, however still no data is being displayed e.g. for s3 buckets/files working fine on the command line.

There is also a javascript error in Chrome

> angular.js:15567 Error: transition superseded
    at $StateProvider.$get (angular-ui-router.js:2827)
    at Object.invoke (angular.js:5141)
    at angular.js:4930
    at getService (angular.js:5084)
    at injectionArgs (angular.js:5109)
    at Object.invoke (angular.js:5133)
    at angular.js:8778
    at forEach (angular.js:387)
    at Object.<anonymous> (angular.js:8776)
    at Object.invoke (angular.js:5141) "Possibly unhandled rejection: {}"
(anonymous) @ angular.js:15567
(anonymous) @ angular.js:11846
processChecks @ angular.js:17975
$digest @ angular.js:19112
$apply @ angular.js:19500
bootstrapApply @ angular.js:1963
invoke @ angular.js:5141
doBootstrap @ angular.js:1961
bootstrap @ angular.js:1981
angularInit @ angular.js:1866
(anonymous) @ angular.js:36430
e @ jquery.min.js:2
t @ jquery.min.js:2
setTimeout (async)
(anonymous) @ jquery.min.js:2
c @ jquery.min.js:2
fireWith @ jquery.min.js:2
fire @ jquery.min.js:2
c @ jquery.min.js:2
fireWith @ jquery.min.js:2
ready @ jquery.min.js:2
B @ jquery.min.js:2
angular.js:15567 Error: transition prevented
    at $StateProvider.$get (angular-ui-router.js:2828)
    at Object.invoke (angular.js:5141)
    at angular.js:4930
    at getService (angular.js:5084)
    at injectionArgs (angular.js:5109)
    at Object.invoke (angular.js:5133)
    at angular.js:8778
    at forEach (angular.js:387)
    at Object.<anonymous> (angular.js:8776)
    at Object.invoke (angular.js:5141) "Possibly unhandled rejection: {}"
(anonymous) @ angular.js:15567
(anonymous) @ angular.js:11846
processChecks @ angular.js:17975
$digest @ angular.js:19112
$apply @ angular.js:19500
bootstrapApply @ angular.js:1963
invoke @ angular.js:5141
doBootstrap @ angular.js:1961
bootstrap @ angular.js:1981
angularInit @ angular.js:1866
(anonymous) @ angular.js:36430
e @ jquery.min.js:2
t @ jquery.min.js:2
setTimeout (async)
(anonymous) @ jquery.min.js:2
c @ jquery.min.js:2
fireWith @ jquery.min.js:2
fire @ jquery.min.js:2
c @ jquery.min.js:2
fireWith @ jquery.min.js:2
ready @ jquery.min.js:2
B @ jquery.min.js:2
angular.js:15567 Error: transition aborted
    at $StateProvider.$get (angular-ui-router.js:2829)
    at Object.invoke (angular.js:5141)
    at angular.js:4930
    at getService (angular.js:5084)
    at injectionArgs (angular.js:5109)
    at Object.invoke (angular.js:5133)
    at angular.js:8778
    at forEach (angular.js:387)
    at Object.<anonymous> (angular.js:8776)
    at Object.invoke (angular.js:5141) "Possibly unhandled rejection: {}"
(anonymous) @ angular.js:15567
(anonymous) @ angular.js:11846
processChecks @ angular.js:17975
$digest @ angular.js:19112
$apply @ angular.js:19500
bootstrapApply @ angular.js:1963
invoke @ angular.js:5141
doBootstrap @ angular.js:1961
bootstrap @ angular.js:1981
angularInit @ angular.js:1866
(anonymous) @ angular.js:36430
e @ jquery.min.js:2
t @ jquery.min.js:2
setTimeout (async)
(anonymous) @ jquery.min.js:2
c @ jquery.min.js:2
fireWith @ jquery.min.js:2
fire @ jquery.min.js:2
c @ jquery.min.js:2
fireWith @ jquery.min.js:2
ready @ jquery.min.js:2
B @ jquery.min.js:2
angular.js:15567 Error: transition failed
    at $StateProvider.$get (angular-ui-router.js:2830)
    at Object.invoke (angular.js:5141)
    at angular.js:4930
    at getService (angular.js:5084)
    at injectionArgs (angular.js:5109)
    at Object.invoke (angular.js:5133)
    at angular.js:8778
    at forEach (angular.js:387)
    at Object.<anonymous> (angular.js:8776)
    at Object.invoke (angular.js:5141) "Possibly unhandled rejection: {}"`

--- a/localstack/utils/bootstrap.py
+++ b/localstack/utils/bootstrap.py
@@ -210,6 +210,13 @@ def canonicalize_api_names(apis=None):
     return apis
 
 
+def is_api_enabled(api):
+    apis = canonicalize_api_names()
+    for a in apis:
+        if a == api or a.startswith('%s:' % api):
+            return True
+
+
 def start_infra_locally():
     bootstrap_installation()
     from localstack.services import infra

--- a/localstack/utils/common.py
+++ b/localstack/utils/common.py
@@ -41,7 +41,7 @@ TMP_PROCESSES = []
 # cache clean variables
 CACHE_CLEAN_TIMEOUT = 60 * 5
 CACHE_MAX_AGE = 60 * 60
-CACHE_FILE_PATTERN = os.path.join(tempfile.gettempdir(), 'cache.*.json')
+CACHE_FILE_PATTERN = os.path.join(tempfile.gettempdir(), '_random_dir_', 'cache.*.json')
 last_cache_clean_time = {'time': 0}
 mutex_clean = threading.Semaphore(1)
 mutex_popen = threading.Semaphore(1)
@@ -886,6 +886,7 @@ def run(cmd, cache_duration_secs=0, print_error=True, asynchronous=False, stdin=
 
     hash = md5(cmd)
     cache_file = CACHE_FILE_PATTERN.replace('*', hash)
+    mkdir(os.path.dirname(CACHE_FILE_PATTERN))
     if os.path.isfile(cache_file):
         # check file age
         mod_time = os.path.getmtime(cache_file)
@@ -999,3 +1000,7 @@ def isoformat_milliseconds(t):
         return t.isoformat(timespec='milliseconds')
     except TypeError:
         return t.isoformat()[:-3]
+
+
+# Code that requires util functions from above
+CACHE_FILE_PATTERN = CACHE_FILE_PATTERN.replace('_random_dir_', short_uid())

